deploy/01-deploy-account-setup.sh

```
@@ -112,6 +113,25 @@ aws cloudformation deploy \
+# ============================================================================
+# Static assets stack
+# ============================================================================
+# Plot asset bucket + the GitHub OIDC role nlm-ckn's publish workflow assumes.
+# Deployed after bootstrap because it references the OIDC provider that stack
+# owns. CAPABILITY_NAMED_IAM: the role has an explicit RoleName.
+echo -e "${GREEN}==> Deploying Static Assets Stack${NC}"
+echo ""
+
+aws cloudformation deploy \
+  --template-file shared/cloudformation/static-assets.yaml \
+  --stack-name ${PROJECT_NAME}-static-assets \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)